source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -141,6 +141,8 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
       route_config_provider_manager_(route_config_provider_manager),
       http2_settings_(Http::Utility::parseHttp2Settings(config.http2_protocol_options())),
       http1_settings_(Http::Utility::parseHttp1Settings(config.http_protocol_options())),
+      max_request_headers_size_(
+          PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, max_request_headers_size_kb, 60)),
```

Comment:
Everywhere that you hard code "60" can you add that as a default constant somewhere and then use it in all the places in this PR?

Reply:
Sure. That'll definitely make it easier if I change the limits too. Would a defaultMaxHeaders var in conn_manager_config.h make sense?

Reply:
As in, would that be the most appropriate place for this. And should each subclass of connmanagerconfig be forced to define it separately? (I wish proto3 had default params...)

Reply:
done